ymir/command/mir/tools/annotations.py

```diff
@@ -56,15 +58,43 @@ def import_annotations(mir_metadatas: mirpb.MirMetadatas, mir_annotation: mirpb.
     """
     unknown_types_and_count: Dict[str, int] = defaultdict(int)
 
-    if not in_sha1_file:
-        logging.error("empty sha1_file")
-        return MirCode.RC_CMD_INVALID_ARGS, unknown_types_and_count
-
     # read type_id_name_dict and type_name_id_dict
     class_type_manager = class_ids.ClassIdManager(mir_root=mir_root)
     logging.info("loaded type id and names: %d", class_type_manager.size())
 
-    image_annotations = mir_annotation.task_annotations[task_id].image_annotations
+    if in_sha1_file:
+        logging.info("wrting annotation in {annotations_dir_path}")
```

Comment:
f"wrting annotation in {annotations_dir_path}"

Reply:
done